### DISCUSSION TITLE
Kids voices not recognized (high pitch)

### DISCUSSION THREAD

**Discussion by jermlab** [speechly/speechly#279] [General]
When testing a new web-based game for kids, their voices are not recognized. If I tell them to speak in a monster voice, it works though. 
Is this due to a high-pass type filter or lack of high-pitched reference data?

**Comment by bigdatabaracus** [comment]
Hello @jermlab 👋  thank you for reporting. Based on our experience the recognition accuracy should not really degrade too much for school-aged children and older. However, for younger children, something like 2-6 years, unfortunately, most likely you will see degradation with our free tier off-the-self models. This is due to very little data in our general training sets covering those. I'm assuming we are taking younger than school-age kids here? 

For [paying customers](https://www.speechly.com/pricing) we are, however, able to train custom models that perform well even for younger children, but unfortunately this does not extend to our free tier users. I encourage you to contact [hello@speechly.com](mailto:hello@speechly.com) if you'd like to hear more about those options.

**Comment by jermlab** [comment]
That is correct. 3-5 years old.
OK, I will inquire about the paid plans.
